src/Pages/miyoungshin/Main/Main.js

```diff
@@ -1,8 +1,74 @@
 import React from 'react';
+import CommonNav from '../../../Components/Nav/Nav';
+import HandleFeed from './Component/FeedList';
+import FEED from './Data/FeedData';
+import './Main.scss';
 
 class MainMi extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      feedsList: [],
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      feedsList: FEED,
+    });
+  }
+
   render() {
-    return <div></div>;
+    return (
+      <div className="MainMi">
+        <CommonNav />
+        <section className="mainSection">
+          <article>
+            <div className="stories">
+              <div class="story">
+                <div class="storyBg">
+                  <img
+                    class="storyProfile"
+                    src="/images/miyoungshin/main/profile/dlwlrma-profile.jpeg"
+                    alt="dlwlrma profile"
+                  />
+                </div>
+                <div class="storyId">dlwlrma</div>
+              </div>
+            </div>
+            {this.state.feedsList.length > 0 && (
+              <HandleFeed feed={this.state.feedsList} />
+            )}
```

Comment:
```suggestion
            {!!feedsList.length && <HandleFeed feed={feedsList} />}
```
반대 연산자를 두번 써서 `!!`를 붙이면 어떤 값이든 불리언 값으로 환원되기 때문에 위와 같이 써주는 것도 가능할 것 같습니다

Reply:
이번에 새롭게 알게 되었습니다 😄 감사합니다.